doc/getstarted/build_and_install/build_from_source_en.md

```diff
@@ -162,16 +162,27 @@ mkdir build && cd build
 
 Finally, you can build and install PaddlePaddle:
 
-```bash
-# you can add build option here, such as:    
-cmake .. -DCMAKE_INSTALL_PREFIX=<path to install>
-# please use sudo make install, if you want to install PaddlePaddle into the system
-make -j `nproc` && make install
-# set PaddlePaddle installation path in ~/.bashrc
-export PATH=<path to install>/bin:$PATH
-# install PaddlePaddle Python modules.
-sudo pip install <path to install>/opt/paddle/share/wheels/*.whl
-```
+- Install PaddlePaddle with `make install`
```

Comment:
I think we can use `pip install` instead of `make install`, so I deleted the `make install`